lib/boost_cpu/boost_cpu.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <logging/log.h>
+#include <zephyr.h>
+#include <nrfx_clock.h>
+
+#if CONFIG_BOOST_CPU_CLOCK_AT_STARTUP
+#include <init.h>
+#endif
+
+LOG_MODULE_REGISTER(boost_cpu, CONFIG_BOOST_CPU_LOG_LEVEL);
+
+void cpu_boost(bool enable)
```

Comment:
How about `enum` or `int` to potentially reuse this API for CPUs with more than two operationg points?